.gitattributes

```diff
@@ -1,2 +1,2 @@
 # Some storm-webapp logviewer tests require input files to have LF line endings due to byte counting.
-storm-webapp/src/test/resources/*.log.test text eol=lf
```

Comment:
Going by the git documentation I'm pretty sure this line should still be here. We want to force LF on checkout for these files, even if the user is on Windows and checking out with `core.autocrlf=true`. `text=auto` seems to only force LF on check in.

Reply:
Agreed. I thought that this would be covered, but it falls under ntars anyway.